src/components/PickerItem/PickerItem.tsx

```diff
@@ -10,14 +10,14 @@ import styles from './styles';
 
 interface IPickerItemProps {
   country: ICountry;
-  onCountrySelect: (code: string) => void;
+  onCountrySelect: (selectedCountry: ICountry) => void;
   textStyle?: StyleProp<TextStyle>;
 }
 
 const PickerItem: React.FC<IPickerItemProps> = ({ country, onCountrySelect, textStyle }) => {
-  const { name, flag, callingCode, alpha2Code } = country;
+  const { name, flag, callingCode } = country;
   return (
-    <TouchableOpacity onPress={() => onCountrySelect(alpha2Code)}>
+    <TouchableOpacity onPress={() => onCountrySelect(country)}>
```

Comment:
returning a country object from this because i can return the calling code to the user and handle the filter process with alpha2Code